SwiftyCollection.podspec

```diff
@@ -0,0 +1,34 @@
+#
+# Be sure to run `pod lib lint SwiftyCollection.podspec' to ensure this is a
+# valid spec before submitting.
+#
+# Any lines starting with a # are optional, but their use is encouraged
+# To learn more about a Podspec see https://guides.cocoapods.org/syntax/podspec.html
+#
+
+Pod::Spec.new do |s|
+  s.name             = 'SwiftyCollection'
+  s.version          = '0.1.0'
+  s.summary          = 'Safe operations on collections'
+
+# This description is used to generate tags and improve search results.
+#   * Think: What does it do? Why did you write it? What is the focus?
+#   * Try to keep it short, snappy and to the point.
+#   * Write the description between the DESC delimiters below.
+#   * Finally, don't worry about the indent, CocoaPods strips it!
+
+  s.description      = <<-DESC
+  This library provides functions that give a safe operations to collections and avoid crashes
```

Comment:
I guess this could be a little more explicit. Plus there is an extra 'a'. Which operations? What is meant by safe?

Reply:
any suggestion?

Reply:
This library provides an extension to the family of Collection protocols (Collection, MutableCollection, and RangeReplacableCollection) for easier and range-safe access to its elements. Most of the added functions return optionals where .none is returned instead of conventional error throwing. A summary of the provided functionality is: element(at:), removeLast, removeFirst, replaceOccurrences, removeElement, and removeDuplicates (see the README file for detailed, complete descriptions.

Reply:
Thank you, I'll update with Readme examples as well